docs/usage.md

```diff
@@ -428,7 +429,8 @@ spec:
       value: minioadmin
     - name: AWS_SECRET_ACCESS_KEY
       value: minioadmin
-
+    - name: AWS_REGION
```

Comment:
Thanks for the change. Nice!

But, basically, I would like to guide users to use the `bucketConfig.region`.
Could you change this example to use `bucketConfig.region`.

Reply:
I updated is for this change request.